lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -633,46 +637,56 @@ void SessionStateEnforcer::prepare_response_to_access(
       config.rat_specific_context.m5gsm_session_context()
           .procedure_trans_identity());
 
-  /* AMBR value need to compared from AMF and PCF, then fill the required
-   * values and sent to AMF.
+  /* If there are any static rules configured from Orc8r via NMS it's
+   * corresponding qos info will be send to AMF.
    */
-  // For now its default QOS, AMBR has default values
-  rsp->mutable_session_ambr()->set_br_unit(
-      config.rat_specific_context.m5gsm_session_context()
-          .default_ambr()
-          .br_unit());
-  rsp->mutable_session_ambr()->set_max_bandwidth_ul(
-      config.rat_specific_context.m5gsm_session_context()
-          .default_ambr()
-          .max_bandwidth_ul());
-  rsp->mutable_session_ambr()->set_max_bandwidth_dl(
-      config.rat_specific_context.m5gsm_session_context()
-          .default_ambr()
-          .max_bandwidth_dl());
-  /* This flag is used for sending defult qos value or getting from policy
-   *  value to AMF.
-   */
-  bool flag_set = false;
   for (auto& val : pending_activation) {
-    if (val.rule.qos().max_req_bw_dl() && val.rule.qos().max_req_bw_ul()) {
+    if (val.rule.has_qos()) {
       MLOG(MDEBUG) << "value set for pending_activation"
                    << val.rule.qos().max_req_bw_ul();
-      rsp->mutable_session_ambr()->set_max_bandwidth_dl(
-          val.rule.qos().max_req_bw_dl());
-      rsp->mutable_session_ambr()->set_max_bandwidth_ul(
-          val.rule.qos().max_req_bw_ul());
       rsp->mutable_qos()->CopyFrom(val.rule.qos());
-      flag_set = true;
-      break;
     }
   }
-  if (!flag_set) {
-    auto* convg_qos = rsp->mutable_qos();
-    convg_qos->set_qci(FlowQos_Qci_QCI_9);
-    convg_qos->mutable_arp()->set_pre_vulnerability(
-        QosArp_PreVul_PRE_VUL_ENABLED);
-    convg_qos->mutable_arp()->set_pre_capability(QosArp_PreCap_PRE_CAP_ENABLED);
-    convg_qos->mutable_arp()->set_priority_level(1);
+  /* AMBR value need to compared from AMF and PCF, then fill the required
+   * values and sent to AMF. Otherwise sessiond will set default AMBR info.
+   */

```

Comment:
sorry I think I asked this before, but where does AMF vs PCF comparison happens? I mean is there any line of code showing that